lib/ts/recipe/dashboard/api/user/create/passwordlessUser.ts

```diff
@@ -0,0 +1,95 @@
+import { APIInterface, APIOptions } from "../../../types";
+import STError from "../../../../../error";
+import Passwordless from "../../../../passwordless";
+import PasswordlessRecipe from "../../../../passwordless/recipe";
+import { User } from "../../../../../types";
+import RecipeUserId from "../../../../../recipeUserId";
+import { parsePhoneNumber } from "libphonenumber-js/max";
+
+type Response =
+    | {
+          status: string;
+          createdNewRecipeUser: boolean;
```

Comment:
what will the frontend show in case (createdNewRecipeUser === false || (createdNewRecipeUser === true && user.loginMethods.length === 1))? It means it didn't create a new user (sign in) or account linking happened.

Same goes for emailpassword user creation too

Reply:
it says the user with this email/phone already exists.

Reply:
when createdNewRecipeUser === false on frontend we show a message say `user already exists` and when the accounts are linked we open the primary user details page in new tab.